tests/20-SideMenu/BoardsMenuSection.test.tsx

```diff
@@ -26,62 +27,77 @@ describe("Regular", () => {
   test("Renders section with board menu items", async () => {
     render(<Regular />);
   
-    //TODO: fill this
+    expect(screen.getByText(Regular!.args!.title!)).toBeVisible;
+
+    const boards = screen.getAllByRole("link");
+    expect(boards).toHaveLength((Regular.args as BoardsSectionProps).boards!.length);
+    boards.forEach((board, i) => {
+      expect(board).toHaveTextContent(`!${(Regular.args as BoardsSectionProps).boards[i].slug}`);
+    });
   });
   
-  //Not sure if this should be it's own test or part of the previous test
   test("Board menu items link to boards", async () => {
     render(<Regular />);
-  
-    //TODO: fill this
+    const actionReturn = jest.fn();
+    mocked(action).mockReturnValue(actionReturn);
+    
+    const boards = screen.getAllByRole("link");
+    for (const board of boards) {
+      fireEvent.click(board);
+      await waitFor(() => {
+        expect(action).toBeCalledWith("#slug");
+      });
+    };
   });
   
   test("Correctly renders board without updates", async () => {
```

Comment:
You should add the correct aria-labels to `BoardMenuItem` so that screen readers know whether they have updates or not, and whether those updates are outdated. Each of these tests should then ensure that only the correct state is being displayed.

Reply:
Isn't that what I've done? 
The aria-labels on the `BoardMenuItem`s are:
"BoardSlug" by itself for no updates
"BoardSlug has new updates" for updates that are not outdated
"BoardSlug has updates" for outdated updates (if you think this is not clear enough I can change it to "has outdated updates" but I don't think that's more clear that just the difference between "new updates" and "updates" and adds a bunch more syllables to be read.
The tests are using exact full string matching by default, so my understanding is that they should pass only if only the correct state is being displayed.